backend/src/common/enums/db/model-alias.enum.ts

```diff
@@ -0,0 +1,13 @@
+enum ModelAlias {
+    APPOINTMENTS = 'appointments',
+    CLINIC = 'clinic',
+    DIAGNOSIS = 'diagnosis',
+    DOCTORS = 'doctors',
+    DOCTOR = 'doctor',
+    DOCUMENT = 'document',
+    MESSAGES = 'messages',
+    NOTIFICATIONS = 'notifications',
+    USER = 'user'
+  }
```

Comment:
```suggestion
enum ModelAlias {
    APPOINTMENTS = 'appointments',
    CLINIC = 'clinic',
    DIAGNOSIS = 'diagnosis',
    DOCTORS = 'doctors',
    DOCTOR = 'doctor',
    DOCUMENT = 'document',
    MESSAGES = 'messages',
    NOTIFICATIONS = 'notifications',
    USER = 'user'
  }
```
Remove keys that you don't use for now
For ex: I don't see that you are using `ModelAlias.CLINICS` in the app